app/src/androidTest/java/de/test/antennapod/ui/PreferencesTest.java

```diff
@@ -53,18 +59,24 @@
     private Resources res;
 
     @Rule
-    public ActivityTestRule<PreferenceActivity> mActivityRule = new ActivityTestRule<>(PreferenceActivity.class, false, false);
+    public ActivityTestRule<PreferenceActivity> activityTestRule;
+
+    {
+        activityTestRule = new ActivityTestRule<>(PreferenceActivity.class,
```

Comment:
We don't use that syntax anywhere else. Could you please just use a line break instead of the block here?

Reply:
Sure. I think I was trying to resolve one of the annoying checkstyle errors. I'll change it.

Reply:
Oh, sorry. I didn't realize that the strange syntax with braces was there before. Anyway, would be great if you could make it a normal variable assignment while you already touched the lines here :)